index.html

```diff
@@ -458,6 +469,160 @@ <h4>
           </dl>
         </section>
       </section>
+      <section>
+        <h2>
+          Showing the payment handler
```

Comment:
```suggestion
          Selecting the payment handler
```

Reply:
I advise against the word "show" this is not about User Agent display.

Reply:
Debatable because we literally get here from `.show()`, but yeah.

Reply:
I think that the selection algorithm may be applicable for more scenarios than just show().

Reply:
Potentially, yes.